ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -613,15 +624,13 @@ static int
 azure_file_size(WT_FILE_HANDLE *file_handle, WT_SESSION *session, wt_off_t *sizep)
 {
     azure_file_handle *azure_fh = reinterpret_cast<azure_file_handle *>(file_handle);
-    int ret;
+    WT_DECL_RET;
```

Comment:
I suggest declaring the return value as `int ret`. I don't see any reason to use a macro here (unless I'm missing something?)

Reply:
resolved https://github.com/wiredtiger/wiredtiger/pull/8804/commits/66d81186597758a131f6e70dc2835e8690a63718